packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -171,6 +226,8 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       selectedItems: selectedItems,
       focusedItemIndices: focusedItemIndices,
       onItemsRemoved: _onRemoveSelectedItems,
+      dragDropHelper: dragDropHelper,
+      dragDropEvents: props.dragDropEvents ? props.dragDropEvents : defaultDragDropEvents,
```

Comment:
i think to begin with this seems okay.. if later on a case arises where we want to only override a specific functionality.. we can think about making this more granular